tools/packaging/deb/istio.mk

```diff
@@ -32,6 +35,13 @@ $(foreach DEST,$(ISTIO_DEB_DEST),\
 
 SIDECAR_FILES+=${REPO_ROOT}/tools/packaging/common/envoy_bootstrap.json=/var/lib/istio/envoy/envoy_bootstrap_tmpl.json
 
+ISTIO_EXTENSIONS:=stats-filter.wasm \
```

Comment:
just to be certain: our existing tests now cover this bit of VM distribution?

Reply:
yes, they do. That is how I found that I needed to fix this.